pycaption/scc/__init__.py

```diff
@@ -415,7 +395,12 @@ def _translate_command(self, word):
         # display pop_on buffer [End Of Caption]
         elif word == '942f':
             self.time = self.time_translator.get_time()
-            self.caption_stash.create_and_store(self.buffer, self.time)
+            if self.pop_ons_queue:
```

Comment:
Please add a comment here as it's a fix in case 942c didn't store the pop on queue

Reply:
Added in ac7fe7af4ac9df3db21ae7035d232e7e56a9a525.